docs/advertising/index.rst

```diff
@@ -22,8 +22,9 @@ that respects user privacy.
 
 We recognize that advertising is not for everyone.
 You may :ref:`opt out of paid advertising <advertising/ethical-advertising:Opting Out>`
--- you will still see :ref:`community ads <advertising/ethical-advertising:Community Ads>` --
-or you can go ad-free by `becoming a Gold Member`_ or a `Supporter`_ of Read the Docs.
+although you will still see :ref:`community ads <advertising/ethical-advertising:Community Ads>`.
+You can go ad-free by `becoming a Gold Member`_ or a `Supporter`_ of Read the Docs.
```

Comment:
I think this PR should also make a publicly accessible landing page for Gold. We really need a way to link to it for people that are logged out, and it seems like the most obvious missing piece currently to making it more "understandable".

Reply:
My plan was to repurpose https://readthedocs.org/sustainability/ and go into Gold in more details there.